administration/logging.md

```diff
@@ -91,27 +91,29 @@ Note that the log levels set using the `log:set` commands are not persistent and
 
 ## Create Log Entries in Rules
 
-It is also possible to create own log entries in rules. This is especially useful for debugging purposes.
+There are times, especially when troubleshooting, when it can be helpful to create special rules that log specific variables or the State of Items.
+When using log entries in this way, it is important to set the appropriate log level in your rule.
```

Comment:
I wouldn't say it's important. It is possible/ an option to do that and should be utilized...

Reply:
It can be a help to decide between info and warning for having a different log color/design.
But its not `important`, since the only one who is interested in a appropiate logging is the rule creator self.

Reply:
Good - both comments addressed.